elasticsearch-plugins/resources/elasticsearch-plugin-batch.json

```diff
@@ -0,0 +1,3 @@
+{
+  "parents" : [ "cdap-etl-batch[0,100]" ]
```

Comment:
All the examples in cdap-guides/cdap-etl-adapter-guide depend upon plugins that are shipped with cdap and those already have the correct parent version.